static/src/js/reducers/__tests__/dataQualitySummaries.test.js

```diff
@@ -19,9 +19,9 @@ describe('SET_DATA_QUALITY_SUMMARIES', () => {
         dataQualitySummaries: [{
           id: '1234-ABCD-5678-EFGH-91011',
           name: 'EDSC',
-          provider_id: 'EDSC-TEST',
-          summary: 'data quality summary',
-          updated_at: '2011-01-31T20:42:08Z'
+          // provider_id: 'EDSC-TEST',
```

Comment:
Can the commented out sections be removed? Or will be restored?

Reply:
Removed we were not using it. I've been using graphql so long I forgot that under legacy services we don't get to pick the fields we want. These were not being used but, were being retrieved from the `legacy services` call